heron/common/src/java/com/twitter/heron/common/config/SystemConfigKey.java

```diff
@@ -0,0 +1,396 @@
+//  Copyright 2017 Twitter. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+package com.twitter.heron.common.config;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import com.twitter.heron.common.basics.ByteAmount;
+
+/**
+ * Keys to be used in the SystemConfigs
+ */
+// TODO(mfu): Provide default values for all configs uniformly
+// TODO(mfu): https://github.com/twitter/heron/issues/970
```

Comment:
We now have a mechanism for defaults and we use it for 2 config keys. @maosongfu do you think that's enough for #970 or should we also be adding defaults for other key values. You comment was initially above keys that now have a default, so maybe we're ok now?